test/fixtures/tags.yml

```diff
@@ -6,12 +6,12 @@
 #
 one: {
  name: 'cfl',
- spectrum_id: 1,
+ spectrum_id: 298486374,
```

Comment:
Are you sure this id is static?

Reply:
You're right it's not. Thanks for pointing out, any suggestions to make the fixtures consistent without constant spectrum id?

Reply:
Get the last spectrum instance created and access it with `Spectrum.last`, or something like that. You have to make sure the the Spectrum fixtures is run first than the tags fixtures.

Reply:
I think it's easier to just set static the spectrum ids if it doesn't break other tests

Reply:
On a second thought, are they static, these numbers seem to repeat for most models (sets,comments,spectra...)